pipelines/coaddQualityCore.yaml

```diff
@@ -16,8 +16,8 @@ tasks:
       atools.xPerpCModel: XPerpCModel
       atools.yPerpPSF: YPerpPSF
       atools.yPerpCModel: YPerpCModel
-      atools.skyObjectSkyPlot: SkyObjectSkyPlot
-      atools.skyObjectHistPlot: SkyObjectHistPlot
+      atools.skyObjectSky: SkyObjectSkyPlot
+      atools.skyObjectFlux: SkyObjectHistPlot
```

Comment:
I especially like that we now have information on the quantity that is being plotted.